### DISCUSSION TITLE
too many files loaded at page refresh?

### DISCUSSION THREAD

**Discussion by uplight-dev** [vikejs/vike#846] [Discussion]
Hi

In the examples/vue-full, in dev mode, when browser is refreshed there's a huge number of requests to load various js files or page chunks (as in screenshot).

Is this normal and is there a purpose for it? 
Is there a way to control it?

![image](https://user-images.githubusercontent.com/12542203/235669244-9e634071-ac9b-41bc-a00e-9748e2aff04b.png)

Thanks!

**Comment by brillout** [comment]
Yes that's normal. That's how Vite works.

**Comment by uplight-dev** [comment]
While debugging I suspected this is invoked by loadPageCode from vite-plugin-ssr.
Isn't it the case?

And if it's not configurable, wouldn't this actually slow down the website, if there's network latency involved?
And if I'd build a production version of the site, would this go away?
Thanks!
